tests/utils/resource_utils.py

```diff
@@ -27,11 +28,22 @@ def object_from_json_relative_path(relative_path):
     return object_from_json(get_resource_content(relative_path))
 
 
-def get_deserialized_object_from_json(relative_path, return_type):
-    resp = namedtuple("MockResp", ["status", "reason"])(200, "")
-    rest_resp = RESTResponse(resp, get_resource_content(relative_path))
+def get_deserialized_object_from_resource(return_type, resource_path):
+    payload = get_resource_content(resource_path)
+    return deserialize_object(return_type, payload)
 
-    return ApiClient().deserialize(rest_resp, (return_type,), True)
+
+def deserialize_object(model, payload):
+    """Deserializes the passed payload to an instance of the specified model
+    Disregards unknown fields is
+
+    :param model: OpenApi generated model
+    :param payload: json payload to be deserialized
+    :return: Instance of the model
+    """
+    response = namedtuple("MockResp", ["status", "reason"])(200, "")
+    response = RESTResponse(response, payload)
+    return ApiClient(configuration=Configuration(discard_unknown_keys=True)).deserialize(response, (model,), True)
```

Comment:
do we need a full APIClient to do the deserialization?

Reply:
For sure, we don't. Will correct it.

Reply:
I used the underlying method used in ApiClient(). deserialize()